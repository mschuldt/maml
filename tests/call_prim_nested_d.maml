arduino = Arduino(desktop=True)

@block(once)
def func():
    print(test_add(test_add(1,2), test_add(3,4)))
    print(test_add(test_add(test_add(test_add(1,2), test_add(3,4)),2),
                   test_add(test_add(test_add(1,2), test_add(3,4)),4)))

    print(test_add(test_add(test_add(test_add(test_add(test_add(test_add(test_add(1,2), test_add(3,4)),2),
                                                         test_add(test_add(test_add(1,2), test_add(3,4)),4)),2), test_add(3,4)),2),
                     test_add(test_add(test_add(1,2), test_add(3,4)),4)))
    die(0)
    
arduino.send(func)
