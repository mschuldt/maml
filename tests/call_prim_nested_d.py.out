Initializing avm...
Ready.

ERROR: unrecognized bytecode: '24'
10
26
51
bye.
